pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -121,12 +121,7 @@ func (rc *reconciler) updateSyncTime() {
 
 func (rc *reconciler) syncStates() {
 	volumesPerNode := rc.actualStateOfWorld.GetAttachedVolumesPerNode()
-	for nodeName, volumes := range volumesPerNode {
-		err := rc.attacherDetacher.VerifyVolumesAreAttached(volumes, nodeName, rc.actualStateOfWorld)
-		if err != nil {
-			glog.Errorf("Error in syncing states for volumes: %v", err)
-		}
-	}
+	rc.attacherDetacher.BulkVerifyVolumes(volumesPerNode, rc.actualStateOfWorld)
```

Comment:
Is that ok we still use VerifyVolumesAreAttached instead of changing to BulkVerifyVolume. For some plugins, it might not be bulk polling. Also this name is used in multiple places which is a little hard to differentiate.